packages/xcursor/xcursor.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Xcursor"
+description: "A pure implementation of Xcursor in Ocaml"
```

Comment:
a synopsis that doesn’t tell anything isn’t really useful. The description you had looks more like a synopsis than a description so you can just make this one the synopsis:
```suggestion
synopsis: "A pure implementation of Xcursor in Ocaml"
```

Reply:
Thanks! I will take a look at some better descriptions and update accordingly.

Reply:
Does the new description fit the criteria?